preupg/utils.py

```diff
@@ -862,3 +848,74 @@ def run_generate(self, xml_file, html_file, old_style=False):
         return ret_val
 
 
+class ModulSetUtils(object):
+
+    @staticmethod
+    def check_property(ini_file_path, key, section):
+        """
+        Checks for key in section if key exists, returns it else raise
+        an exception
+
+        @param {string} ini_file_path
+        @param {string} key
+        @param {string} section
+        @return {string} - key value
+        @throws {EnvironmentError} - if key of section doesn't exist
+        """
+        if not ConfigHelper.has_ini_file_section(ini_file_path, section):
+            raise EnvironmentError("Section {0} is missing inside {1} file is"
+                                   .format(section, ini_file_path))
+        key_value = ConfigHelper.get_preupg_config_file(ini_file_path,
+                                                        key, section)
+        if not key_value:
+            raise EnvironmentError("Key {0} inside {1} section\n"
+                                   "is missing in {2}"
+                                   .format(key, section, ini_file_path))
+        return key_value
+
+    @staticmethod
+    def get_assessment_version(module_set_path):
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2